gradle/init.gradle.kts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+val ktlintVersion = "1.0.0"
+
+initscript {
+    val spotlessVersion = "6.22.0"
+
+    repositories {
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath("com.diffplug.spotless:spotless-plugin-gradle:$spotlessVersion")
+    }
+}
+
+rootProject {
+    subprojects {
+        apply<com.diffplug.gradle.spotless.SpotlessPlugin>()
+        extensions.configure<com.diffplug.gradle.spotless.SpotlessExtension> {
+            kotlin {
+                target("**/*.kt")
+                targetExclude("**/build/**/*.kt")
+                ktlint(ktlintVersion)
+                    .userData(mapOf("android_studio" to "true"))
```

Comment:
The [spotless documentation](https://github.com/diffplug/spotless/tree/main/plugin-gradle#ktlint) says to use `mapOf("android" to "true")`.

I can see in the ktlint documentation that `--android` was deprecated, but according to [this issue](https://github.com/pinterest/ktlint/issues/1982) it looks like we should be using `mapOf("code-style", "android_studio")` or maybe just adding `ktlint_code_style=android_studio` in `.editorconfig`.

Reply:
That's a good spot, I'll move it to `.editorconfig` I think that's cleaner.